packages/Pagination/Pagination.jsx

```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react'
```

Comment:
I'm wondering if this component can also be refactored as a functional component with hooks as part of this PR?

Reply:
I think this is out of scope for now. The main purpose of this PR is to move the functionality closer to the desired design. Refactoring for hooks could increase potential complexity and make the process longer.